centreon-open-tickets/www/modules/centreon-open-tickets/class/automatic.class.php

```
@@ -72,6 +72,275 @@ public function __construct($rule, $centreonPath, $openTicketPath, $centreon, $d
+}
+
+/**
+ * isServiceUnique checks if the service is linked to a single host (not to multiple hosts or to a hostgroup)
+ *
+ * @param int $serviceId the id of the service
+ * @throws CollectionException|ConnectionException|ValueObjectException
+ * @return bool
+ */
+function isServiceUnique(int $serviceId): bool
+{
+    global $db;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$db'. (undefined)

(UnusedLocalVariable)